main.js

```diff
@@ -59,7 +59,12 @@ function startTimer(event) {
 
 function startCountDown() {
   var secondsInput = parseInt(seconds.value - 1);
-  var minutesInput = parseInt(minutes.value);
+  var minutesInput = 0;
+  if (minutes.value === ""){
+    minutesInput = 00;
+  } else {
+    minutesInput = parseInt(minutes.value);
+  }
```

Comment:
It looks like on line 62 `minutesInput` is being defined with a value of 0, but being assigned a new value of 00 if `minutes.value` is empty. Is there a reason why you are reassigning it to 00, and how is that different from 0?